ci-operator/config/open-telemetry/opentelemetry-operator/open-telemetry-opentelemetry-operator-main__ocp-4.14-amd64.yaml

```diff
@@ -0,0 +1,109 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  operator-sdk:
+    name: "4.14"
+    namespace: origin
+    tag: operator-sdk
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- build_args:
+  - name: VERSION_PKG
+    value: github.com/open-telemetry/opentelemetry-operator/internal/version
+  - name: VERSION
+    value: 0.79.0-20-g10fbf9f
+  - name: VERSION_DATE
+    value: "2023-06-30T09:14:14Z"
+  - name: OTELCOL_VERSION
+    value: 0.80.0
+  - name: TARGETALLOCATOR_VERSION
+    value: 0.80.0
+  - name: OPERATOR_OPAMP_BRIDGE_VERSION
+    value: 0.80.0
+  - name: AUTO_INSTRUMENTATION_JAVA_VERSION
+    value: 1.26.0
+  - name: AUTO_INSTRUMENTATION_NODEJS_VERSION
+    value: 0.40.0
+  - name: AUTO_INSTRUMENTATION_PYTHON_VERSION
+    value: 0.39b0
+  - name: AUTO_INSTRUMENTATION_DOTNET_VERSION
+    value: 0.7.0
+  - name: AUTO_INSTRUMENTATION_GO_VERSION
+    value: v0.2.1-alpha
+  - name: AUTO_INSTRUMENTATION_APACHE_HTTPD_VERSION
+    value: 1.0.3
+  context_dir: .
+  dockerfile_path: Dockerfile
+  to: opentelemetry-operator
+- context_dir: .
+  dockerfile_path: ./tests/e2e-openshift/Dockerfile
+  to: opentelemetry-tests-runner
+operator:
+  bundles:
+  - as: opentelemetry-bundle
+    context_dir: .
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:0.80.0
+    with: pipeline:opentelemetry-operator
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: opentelemetry-e2e-tests
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
```

Comment:
Can we use a different cluster_profile for this upstream pre-submit testing. Our team's aws-cspi-qe cluster_profile is only budgeted to run periodic testing once a week.